lib/ckeditor/dimagi-README.md

```diff
@@ -6,7 +6,6 @@ Go to the [CK build editor](http://ckeditor.com/builder).
 Use the following plugins:
 * clipboard (for drag and drop)
 * entities
-* floatingspace
```

Comment:
Good point. 
Not necessary as it is an internal dependency for clipboard which gets automatically included while using the online builder.
However, it would be useful to have a list of sub dependencies (there are others too) when building from source. 
**Will be including them in a sub section.**